File: 001 - Multiples of 3 and 5/1.c
#include<stdio.h>
main(){

int a,i;

for(a=1; a<1000; a++){

if(a%3==0 || a%5==0){

i = i+a;

}
}
printf("Sum of all the multiples of 3 or 5 below 1000 : %d", i);
}

File: project_euler/001 - Multiples of 3 and 5/1.py
sum = 0
for i in range(1,1000):
    if i%3==0:      #Check whether i is divisible by 3
        sum+=i
    elif i%5==0:    #Check whether i is divisible by 5
        sum+=i

print "Sum of all the multiples of 3 or 5 below 1000:", sum
